lib/puppet/provider/uri_parsing.rb

```diff
@@ -81,7 +81,8 @@ def special_resources_assign(key)
   when 'associatedUplinkSetUri' then 'UplinkSet'
   when 'parentTaskUri', 'associatedTaskUri' then 'Task'
   when 'nativeNetworkUri' then 'EthernetNetwork'
-  when 'enclosureUris' then 'Enclosure'
+  # Uncomment this line, once array handling is done for enclosureUris
```

Comment:
You can add a TODO: and add the issue number.
Ex: TODO: comments with issue number

Reply:
Done